app/views/home/_administration.html.erb

```diff
@@ -2,7 +2,7 @@
   <h2>Administration</h2>
   <ul>
     <li><%= link_to 'Requests dashboard', admin_index_path %></li>
-    <li><%= link_to "Broadcast messages", "#"%> </li>
+    <li><%= link_to "Broadcast messages", messages_path %> </li>
```

Comment:
Not a blocker, but currently this partial is rendered if the user can manage Requests (because we didn't have anything but the Requests Dashboard link at the time).  The users who can manage requests can also manage messages, so this happens to work now.

I can't think of a good alternative off-the-top of my head, so I'm willing to :shipit: and address this later (if at all),